### PR TITLE
E2e Local Minikube Fixes

### PR DESCRIPTION
**Pull request by ymesika**
Few corrections to the [E2E local test setup](https://github.com/istio/istio/tree/master/tests/e2e/local/minikube) on MacOS with Minkube.

### CODE REVIEW

File: tests/e2e/local/minikube/install_prereqs_macos.sh [hunk 1]
```diff
@@ -92,7 +92,7 @@ minikube --help > /dev/null
 if [[ $? -ne 0 || (`minikube version` != *"minikube version: v0.27.0"*) ]];
 then
     echo "Minikube version 0.27.0 is not installed. Installing it using curl."
-    curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```
Comment (gargnupur) [inline tests/e2e/local/minikube/install_prereqs_macos.sh]: Thanks :)

File: tests/e2e/local/minikube/setup_host.sh [hunk 2]
```diff
@@ -34,6 +34,8 @@ eval $(minikube docker-env)
 
 kubectl get pods -n kube-system | grep kube-proxy |  grep Running > /dev/null
 while [ $? -ne 0 ]; do
+  echo "kube-proxy not ready, will check again in 5 sec"
+  sleep 5
```
[the remark this answers is not in the document]
Comment (ymesika) [reply tests/e2e/local/minikube/setup_host.sh]: You are not suppose to run the script too often so 5 sec vs 1sec of your life isn't such a big annoyance. IMHO.

**Pull request merged**